js/background.js

```diff
@@ -77,7 +77,8 @@
     theme: 'sunburst',
     font: 'Inconsolata',
     fontSize: 'medium',
-    lineNumbers: true
+    lineNumbers: true,
+    languageBlacklist: ''
```

Comment:
I'd generalise this by making the value a regular expression that's matched against the URL. This way we can not only skip specific languages but whole domains. What do you think? I'd call it, `ignore`.